packages/http-server/src/__tests__/integration/http-server.integration.ts

```diff
@@ -208,6 +214,52 @@ describe('HttpServer (integration)', () => {
     expect(server.url).to.equal(`http://127.0.0.1:${server.port}`);
   });
 
+  it('supports HTTP over unix socket', async () => {
+    if (os.platform() === 'win32') return;
+    const socketPath = path.join(os.tmpdir(), 'test.sock');
+    server = new HttpServer(dummyRequestHandler, {
+      path: socketPath,
+    });
+    await server.start();
+    expect(getAddressFamily(server)).to.equal('ipc');
+    expect(server.address).to.eql(socketPath);
+    expect(server.host).to.be.undefined();
+    expect(server.port).to.eql(0);
+    expect(server.url).to.eql('http+unix://' + encodeURIComponent(socketPath));
+    await supertest(server.url)
+      .get('/')
+      .expect(200);
+  });
+
+  it('supports HTTP over Windows named pipe', async () => {
+    if (os.platform() !== 'win32') return;
```

Comment:
It's a really bad practice to skip tests this way. It creates impression that the test verified the functionality and passed, even when nothing was done at all.

A better option is to use `skipIf` helper from `@loopback/testlab`.

```ts
skipIf(os.platform() !== 'win32', it, 'supports HTTP over Windows named pipe', async () => {
  // ...
});
```

@raymondfeng @hacksparrow